schema.graphql

```diff
@@ -176,3 +180,16 @@ type BalancerMetaPoolStrategy @entity {
   rETH: BigInt!
   weth: BigInt!
 }
+
+type ExchangeRate @entity {
+  """
+  Format: 'blockNumber:pair' ex '123456789:ETH_USD'
+  """
+  id: ID!
+  timestamp: DateTime! @index
+  blockNumber: Int! @index
+  pair: String!
```

Comment:
Why is a pair needed if `base` and `quote` exists?

Reply:
Can't entirely say I'm sure it's needed. Maybe we can remove later.